common/experiment_utils.py

```diff
@@ -23,6 +23,12 @@
 CONFIG_DIR = 'config'
 
 
+def get_internal_experiment_config_relative_path():
```

Comment:
is internal important in name ?

Reply:
I think so because I want to distinguish between the internal config file and the external one, which is different.